rotkehlchen/tests/unit/globaldb/test_upgrades.py

```diff
@@ -664,3 +730,280 @@ def test_applying_all_upgrade(globaldb: GlobalDBHandler):
     assert globaldb.get_setting_value('version', 0) == GLOBAL_DB_VERSION
     with globaldb.conn.cursor() as cursor:
         assert cursor.execute('SELECT COUNT(*) from assets WHERE identifier="eip155:/erc20:0x32c6fcC9bC912C4A30cd53D2E606461e44B77AF2"').fetchone()[0] == 1  # noqa: E501
+
+
+def test_update_consistency_with_packaged_db(
```

Comment:
This is too big a function. Can you move it to its own test file with a proper name?